target/linux/ath79/dts/ar9342_ubnt_nanobeam-ac.dts

```diff
@@ -51,11 +50,11 @@
 	status = "okay";
 
 	/* default for ar934x, except for 1000M and 10M */
-	pll-data = <0x06000000 0x00000101 0x00001313>;
```

Comment:
So, is the comment wrong now too?  or was it wrong before?

Reply:
Not sure, I used the values from the  Litebeam AC Gen2, since it is a 1port devices that has a working ethernet switch. The next commit merges all the 1port devices together.